### PR TITLE
Rename to palindrom-bunny

### PR DESCRIPTION
**Pull request by alshakero**
I'm not 100% sure I should bump the version to 2.0.0, since there is no change in the code at all. And the name will change, no information is needed to show that it's a breaking change.

Please rename the repo if decided to merge.

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[0228/155442:ERROR:tcp_listen_socket.cc(76)] Could not bind socket to 127.0.0.1:6004
```
Comment (tomalec) [inline debug.log]: Could you remove this file?

Comment (alshakero) [reply debug.log]: Sure.

**Review by tomalec — CHANGES_REQUESTED** [review]
Please remove `debug.log`, and bump the version, according to semver, we broke backward compatibility by changing the API/ element name, so I think we should do so.

**Comment by alshakero** [comment]
@tomalec done.

**Pull request merged**